test/components/granules/overview.js

```diff
@@ -65,18 +65,14 @@ test('GranulesOverview generates bulkAction for recovery button', function (t) {
   const recoverFilter = (object) => object.text === 'Recover Granule';
   const recoverActionList = listBulkActions.filter(recoverFilter);
   t.is(recoverActionList.length, 1);
-
-  const recoverAction = recoverActionList[0];
-  t.truthy(recoverAction);
-  t.is(recoverAction.text, 'Recover Granule');
 });
 
 test('GranulesOverview does not generate bulkAction for recovery button', function (t) {
   const dispatch = () => {};
   const workflowOptions = [];
   const stats = { count: 0, histogram: {}, stats: {} };
   const location = { pathname: 'granules' };
-  const config = {};
```

Comment:
figure this better reflects the "null state" of an un-configured `recoveryPath`.